recipe/meta.yaml

```diff
@@ -29,25 +30,31 @@ requirements:
     - jinja2 >=3.1.3,<4
     - mashumaro >=3.9,<4.0
     - pathspec >=0.9,<0.13
-    - protobuf >=5.0,<6.0
+    - protobuf >=4.0.0,<5.0.0
```

Comment:
Should we increase the upper bound to <6.0? https://github.com/dbt-labs/dbt-common/blob/main/pyproject.toml#L38

Reply:
Isn't `protobuf` one of those where the pinnings are quite important for interop?  I would have thought we keep it to  within the one major version, ie. 4.* or 5.*.  Do we have any `protobuf` guidelines written down for reference?

Reply:
There's no tag for this release on Github, but the pypi source has `"protobuf>=4.0.0,<5.0.0",`